### PR TITLE
CLDR-15196 common order for grammatical features

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15196

- [ ] This PR completes the ticket.

### CODE REVIEW

File: common/annotations/dsb.xml [hunk 1]
```diff
@@ -832,7 +832,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🧑‍⚕">chórobny sotša | gójc | gójc/gójcowka | strowotnistwo | terapeut</annotation>
 		<annotation cp="🧑‍⚕" type="tts">gójc/gójcowka</annotation>
 		<annotation cp="👨‍⚕">doktor | gójc | gójcojski kitel</annotation>
-		<annotation cp="👨‍⚕" type="tts"/>
+		<annotation cp="👨‍⚕" type="tts">doktor</annotation>
```
Comment (srl295) [inline common/annotations/dsb.xml:835]: Is this the TTS err issue we have been seeing only sometimes?

**Comment by macchiati** [comment]
This is ready for review, Annemarie. I cleaned up the ordering a bit, which you can see from tools/cldr-code/src/main/java/org/unicode/cldr/util/GrammarInfo.java, and also added the reasoning behind the ordering.

(Just noticed a typo in the comments and will fix that.)

**Review by srl295 — APPROVED** [review]
DUgCET - the DUCET for grammar

**Comment by AEApple** [comment]
Can we add a note in the Survey Tool about the ordering of the cases to inform the vetters about the ordering?

**Comment by macchiati** [comment]
Yes, whenever we push to production we need to make sure that we update the
documentation (and send out a note pointing to the change). We'll want to
try it out on smoketest first, of course!

Mark

**Comment by macchiati** [comment]
Yes, common/annotations/dsb.xml
<https://github.com/unicode-org/cldr/pull/1641#discussion_r761416911>: was
for the test failure (other failures are hidden in non-exhaustive mode).

Mark

**Comment by macchiati** [comment]
If someone reviews, then I can start checking it out on Smoketest.

**Review by srl295 — APPROVED** [review]

**Comment by macchiati** [comment]
Gakk, had to squash because of a bad commit message... so asking for another review. (Really wish squashing didn't require rereview.)

**Review by btangmu — APPROVED** [review]

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
> Yes, common/annotations/dsb.xml <[#1641 (comment)](https://github.com/unicode-org/cldr/pull/1641#discussion_r761416911)>: was for the test failure (other failures are hidden in non-exhaustive mode). Mark
> […](#)

*ideally* that would be in the commit message also.

**Comment by macchiati** [comment]
I merged this in and checked it out on Smoketest. Noted some problems:
https://unicode-org.atlassian.net/browse/CLDR-15196?focusedCommentId=163433
Mark

**Pull request merged**